modin/test/test_headers.py

```diff
@@ -29,7 +29,7 @@ def test_headers():
         for file in files:
             filepath = os.path.join(subdir, file)
             if file.endswith(".py") and file not in exclude_files:
-                with open(filepath, "r") as f:
+                with open(filepath, "r", encoding="utf8") as f:
```

Comment:
Why did you add `encoding` parameter?

Reply:
to make it pass on my box where default encoding is `cp-1251`

Reply:
What's the point?.. it unifies the behaviour between Linux and Windows, and everything passes everywhere.
I would rather leave it be as I did it now.

Reply:
Ok, I got it